### PR TITLE
translate 2014/09/18/ruby-2-2-0-preview1-released (zh_cn)

### PR DESCRIPTION
**Pull request by li-thy-um**
ruby-2-2-0-preview1-released
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix

Please help review this pull request.

**Comment by reyesyang** [comment]
:+1:

File: zh_cn/news/_posts/2014-09-18-ruby-2-2-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    * File.birthtime, File#birthtime
+* 依赖库:
+  * 升级 Psych 2.0.6
+  * 升级 Rake 10.3.2+ (e47d0239)
+  * 升级 RDoc 4.2.0.alpha (21b241a)
+  * 升级 RubyGems 2.4.1+ (713ab65)
+  * 升级 test-unit 3.0.1 (从资料库中删除，但捆绑在压缩包中)
+  * 升级 minitest 5.4.1 (从资料库中删除，但捆绑在压缩包中)
+  * 弃用 mathn
+* C API
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-09-18-ruby-2-2-0-preview1-released.md]: Not sure there're spaces here?

Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014-09-18-ruby-2-2-0-preview1-released.md]: In this case, it is correct. Check the english one if you are in doubt.

**Comment by JuanitoFatas** [comment]
Thanks. Merged as 5241b50.

**Pull request closed without merging**